### DISCUSSION TITLE
Excuse me, does WINMERGE support formatting?

### DISCUSSION THREAD

**Discussion by Just-dot** [WinMerge/winmerge#1564] [Q&A]
For example,,I want to format JS, json and other files in WINMERGE

**Comment by Just-dot** [comment]
Sorry, I have found the option to format. but it's not very friendly

**Reply by Pierre-de-la-Verre** [reply]
> Sorry, I have found the option to format. but it's not very friendly

But very powerfull ;-)
Can this issue be closed?
